packages/back-end/types/datasource.d.ts

```diff
@@ -133,6 +133,11 @@ export type DataSourceSettings = {
   // @deprecated
   experimentDimensions?: string[];
   notebookRunQuery?: string;
+  informationSchema?: {
```

Comment:
I don't love having this object live outside of the actual `informationSchema` but it felt more appropriate than creating an empty `informationSchema` record and having a ton of optional types.

Reply:
If you added it to the information schema model instead, would there need to be any optional types?  Couldn't you just have an empty array for databases?  Everything else is relevant (created/updated dates, etc.)